tests/istio_test.go

```diff
@@ -25,9 +25,12 @@
 package tests
 
 import (
+	"github.com/skydive-project/skydive/topology/probes/istio"
 	"testing"
+)
 
-	"github.com/skydive-project/skydive/topology/probes/istio"
+const (
+	b = "https://raw.githubusercontent.com/istio/istio/release-1.0/samples/bookinfo"
```

Comment:
nit: rename b to something more explicit. `bookInfo` ?

Reply:
Otherwise LGTM

Reply:
DONE